runtime/flang/CMakeLists.txt

```diff
@@ -454,25 +470,6 @@ set_property(
  PG_PIC
  )
 
-## CMake does not handle module dependencies between Fortran files,
-## we need to help it
```

Comment:
This isn't needed because of https://github.com/flang-compiler/flang/pull/464/files#diff-af3b638bc2a3e6c650974192a53c7291R26